doc/source/reference/random/parallel.rst

```diff
@@ -97,6 +97,10 @@ streams, about :math:`2^{20}`, then the probability that at least one pair of
 them are identical is about :math:`2^{-88}`, which is in solidly-ignorable
 territory ([2]_).
 
+Note that while `~SeedSequence` attempts to solve many of the issues related to
+user-provided small seeds, we still recommend using ``secrets.randbits(128)`` seeds
+for community practice reasons.
```

Comment:
```suggestion
user-provided small seeds, we still recommend using :py:func:`secrets.randbits` to generate
seeds to avoid biases introduced by human-chosen seeds.
```

I think this is a more understandable distillation of @rkern's comment in the issue, given the rest of the text above.

You could also add a rst link anchor above [this paragraph](https://github.com/numpy/numpy/blob/2da02ea321f557c0cfe0ad6d0e7d8a4354c51103/doc/source/reference/random/index.rst?plain=1#L59) and then add an internal link here to the example in the docs where we introduce the preferred way to generate seeds.

Reply:
You probably want to move this up (or into) the paragraph starting "`~SeedSequence` avoids these problems ...". Down here is after the details about the parallelism and farther away from the claims about conditioning poor-quality seeds.

Reply:
Thanks for the feedback. 
* @ngoldbaum The paragraph you pointed to is the one that I felt most relevant and I'll get a link to it. 
* @rkern Thanks for the placement suggestion. I debated where to place it, and was hoping you'd chime in. Do you think it would be better to put a footnote right after "these problems [1]" and then add it to the other two footnotes at the end of the section?
* I thought the discussion in the issue was a great addition to the page.  Would it be worth putting a gh_25778 link in the docs?